crowd_anki/config/config_settings.py

```diff
@@ -49,17 +49,26 @@ class Properties(Enum):
         EXPORT_CREATE_DECK_SUBDIRECTORY = ConfigEntry("export_create_deck_subdirectory", True)
         IMPORT_NOTES_IGNORE_DECK_MOVEMENT = ConfigEntry("import_notes_ignore_deck_movement", False)
 
-    def __init__(self, addon_manager=None, init_values=None):
+    def __init__(self, addon_manager=None, init_values=None, window=None):
         self.addon_manager = addon_manager or mw.addonManager
         self._config = init_values or addon_manager.getConfig(__name__)
+        self._window = window
```

Comment:
this should be initialized similar to the `addon_manage` (i.e. be assigned `mw` by default) also probably should be a more specific dependency - does `pm` stands for something like `profile_manager`?

Reply:
Do you mean like this? (see the latest commit)